docs/tests/e2e-web.md

```diff
@@ -55,13 +53,43 @@ It is possible to run tests with image snapshots to test for visual regressions.
 When you need to update image snapshots you have 2 options:
 - use CI job. This will generate new snapshots in artifacts together with a handy script that updates your snapshots locally. Check the log output. 
 - use `./docker/docker-suite-snapshots.sh`. This does the same as `./docker/docker-suite-test.sh`, the only difference is it won't fail on non-matching snapshots but generate new snapshots.
-# Old notes (to be removed) 
-
-## Notes on bridge (trezord)
+# Notes
+
+## Running e2e tests locally macOS
+1. Follow first three steps in macOS category at [trezor-user-env](https://github.com/trezor/trezord-go/tree/cypress) repository.
+2. Increase RAM in Docker to at least 4GB (Docker->Preferences->Resources->Advanced) otherwise, the app during tests does not even load.
+3. Use `./docker/docker-suite-install.sh`.
+4. In the command line, set two environment variables:
+``
+export HOSTNAME=`hostname`
+``
+``
+export DISPLAY=${HOSTNAME}:0
+``
+5. Use `./docker/docker-suite-test.sh`.
+-- A Cypress window should open.
+-- Wait until the project is built (a warning about http://localhost:3000/ is not available, should disappear on the retry button click).
+6. Run tests.
+-- Should open a browser window.
+-- If the app in tests is not loading even after two retries. Stop tests, open a new tab, navigate to http://localhost:3000/, refresh the page until the app is loaded. Close the tab and run tests again.
+
+### Troubleshooting
+- `[...ERROR:browser_main_loop.cc(1434)] Unable to open X display.`
+-- Make sure the XQuartz app is launched and you can see its terminal.
+-- Check that environment variables are properly set:
+``
+echo $HOSTNAME # e.g. name.local
+``
+``
+echo $DISPLAY # e.g. name.local:0
+``
+-- Do not mix native command line with command line in Visual Studio Code.
+
+## Bridge - trezord (deprecated note - to be removed)
 Cypress has 2 problems when communicating with trezord. 
 
 1. Does not send request origin headers correctly -> problem with CORS.
-1. Tests in CI fail occasionally on POST to '/' with 403 -> problem with CSFR.
-1. Change default address to 0.0.0.0 instead of 127.0.0.1 to make it work on mac without any other configuration
+2. Tests in CI fail occasionally on POST to '/' with 403 -> problem with CSFR.
+3. Change default address to 0.0.0.0 instead of 127.0.0.1 to make it work on mac without any other configuration
```

Comment:
This was here from @mroz22 right? Because I am not sure this is still needed after https://github.com/trezor/trezor-user-env/commit/eb3ad735a174ecdffada3d7fb8e4ae649dc629cf

(Btw: markdown converts `1., 1., 1.` to the proper numbering but I agree I do not like it either :) )

Reply:
tested and I am going to delete this